spec/lib/miq_automation_engine/service_methods/miq_ae_service_tenant_spec.rb

```diff
@@ -5,12 +5,17 @@ module MiqAeServiceTenantSpec
     let(:settings) { {} }
     let(:tenant) { Tenant.create(:name => 'fred', :domain => 'a.b', :parent => root_tenant, :description => "Krueger") }
 
+    let(:cpu_quota) { TenantQuota.create(:name => "cpu_allocated", :unit => "int", :value => 2, :tenant_id => tenant.id) }
+    let(:storage_quota) { TenantQuota.create(:name => "storage_allocated", :unit => "GB", :value => 160, :tenant_id => tenant.id) }
```

Comment:
@mkanoor Can these be moved into the test where they are used instead of having them defined here and can you use FactoryGirl?  Also, I do not see any point in creating the Service Model instances for these since the `id` value will be the same for both.